osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -228,7 +228,7 @@ public override void Draw(Action<TexturedVertex2D> vertexAction)
                 {
                     var offset = triangle_size * new Vector2(particle.Scale * 0.5f, particle.Scale * 0.866f);
 
-                    var triangle = new Triangle(
+                    Triangle triangle = new Triangle(
```

Comment:
we're creating an instance of `Triangle`, is there a reason not to use `var` here as well?

Reply:
The object being returned is a Triangle, so making it Triangle instead of var makes it stand out.

Reply:
this makes no sense, you're returning a `Triangle`, we would expect `var` to be `Triangle`?